pkg/oc/admin/registry/registry.go

```diff
@@ -147,14 +148,15 @@ const (
 // NewCmdRegistry implements the OpenShift cli registry command
 func NewCmdRegistry(f *clientcmd.Factory, parentName, name string, out, errout io.Writer) *cobra.Command {
 	cfg := &RegistryConfig{
-		ImageTemplate:  variable.NewDefaultImageTemplate(),
-		Name:           "registry",
-		Labels:         defaultLabel,
-		Ports:          strconv.Itoa(defaultPort),
-		Volume:         "/registry",
-		ServiceAccount: "registry",
-		Replicas:       1,
-		EnforceQuota:   false,
+		ImageTemplate:    variable.NewDefaultImageTemplate(),
+		Name:             "registry",
+		Labels:           defaultLabel,
+		Ports:            strconv.Itoa(defaultPort),
+		Volume:           "/registry",
+		ServiceAccount:   "registry",
+		Replicas:         1,
+		EnforceQuota:     false,
+		DeploymentConfig: true,
```

Comment:
to make sure we don't break existing installer flows/etc... the flag can be deprecated in 3.11 when we update installer to use deployment configs. no reason why oc cluster up can't use this by default.